disperser/common/blobstore/blob_metadata_store.go

```diff
@@ -218,6 +220,21 @@ func (s *BlobMetadataStore) GetBlobMetadataInBatch(ctx context.Context, batchHea
 	return metadata, nil
 }
 
+// GetBlobMetadataByAccount Count returns the count of all the metadata with the given status
+// Because this function scans the entire index, it should only be used for status with a limited number of items.
+// It should only be used to filter "Processing" status. To support other status, a streaming version should be implemented.
+func (s *BlobMetadataStore) GetBlobMetadataCountByAccountID(ctx context.Context, accountID core.AccountID) (int32, error) {
```

Comment:
We probably want the total amount of data rather than the total number of blobs. Or perhaps both. 

Let's do another sync on the exact use case for this work!